404.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>404</title>
+  <meta name="template" content="404" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <script src="cclibs/scripts/scripts.js" type="module"></script>
```

Comment:
cclibs is not used in the PR anymore. We are using cc. The script path would need to be changed accordingly.

Reply:
@aishwaryamathuria thanks for confirming. I have updated the path.